akd/src/directory.rs

```diff
@@ -562,8 +571,8 @@ where
     /// the "latest" epoch. There is a "special" flow in the storage layer
     /// to do a storage-layer retrieval which ignores the cache
     ///
-    /// NOTE: Due to the use of std::thread::sleep(.) this will BLOCK
-    /// the polling thread, and should be allocated it's own thread since it won't
+    /// NOTE: Due to the use of [std::thread::sleep] this will BLOCK
```

Comment:
This is no longer accurate right? We're using tokio's sleep which does NOT block the thread, and therefore this can just be `spawn`'d

Reply:
Excellent! Removed the note.